homeassistant/components/airzone_cloud/entity.py

```diff
@@ -58,6 +61,37 @@ def get_airzone_value(self, key: str) -> Any:
         return value
 
 
+class AirzoneWebServerEntity(AirzoneEntity):
+    """Define an Airzone Cloud WebServer entity."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        entry: ConfigEntry,
+        ws_id: str,
+        ws_data: dict[str, Any],
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator)
+
+        self.ws_id = ws_id
+
+        self._attr_device_info = DeviceInfo(
+            connections={(dr.CONNECTION_NETWORK_MAC, ws_id)},
+            identifiers={(DOMAIN, f"{entry.unique_id}_{ws_id}")},
+            manufacturer=MANUFACTURER,
+            name=f"WebServer {ws_id}",
```

Comment:
This ID doesn't seem to be user-friendly / meant to be part of an entity.

Reply:
Yes, I know, but the WebServer ID is displayed in the Master Thermostat and is also used by Airzone to identify specific installation devices (it's also shown in the Airzone app where the WebServer firmware can be upgraded).
And since one installation can have multiple WebServers, as opposed to the local integration in which the WebServer is the one providing the Local API, this was the only way I could think of differentiating them.